pyproject.toml

```diff
@@ -63,6 +63,7 @@ dependencies = [
     'typing-extensions>=4.6.1',
     'annotated-types>=0.4.0',
     "pydantic-core==2.11.0",
+    "importlib_metadata; python_version=='3.7'"
```

Comment:
Do we want to restrict the range of `importlib_metadata` to `>=` something?

Reply:
@Kludex - I don't think so, currently we import it like this:
https://github.com/pydantic/pydantic/blob/47aa97e5311ed40dd1fe70f6cade874b4dd429e5/pydantic/version.py#L24-L29.

Let me know if you have a different approach.

Reply:
It's fine. 👍